porting/first-steps.rst

```diff
@@ -57,7 +57,7 @@ Arch
 ^^^^^^^^^^
 
 If you have a pure ``amd64`` running you need to add ``[multilib]`` repository to your ``/etc/pacman.conf`` . This will allow you to install and run ``i686`` packages. Please refer to `<https://wiki.archlinux.org/index.php/multilib>`_
-Assuming you have already installed ``base-devel`` meta package; if not please install ``base-devel``.
+Assuming you have already installed ``base-devel`` group while installing Arch; if not please install ``base-devel``.
```

Comment:
please remove "while installing Arch", it makes no sense. you can install the group in a year after you've finished installing Arch.. also, add a comma after "if not"